README.md

```diff
@@ -38,6 +38,8 @@ npm run build
 
 The Webpack will output all HTML files to a folder called `dist`.
 
+Turn markdown to html: `tools/build_ai_guide.sh`
```

Comment:
this doesn't apply to the main future website, we should likely call this out in a separate README